lib/membrane_native/raw_video.ex

```diff
@@ -0,0 +1,92 @@
+defmodule Membrane.Native.RawVideo do
+  @moduledoc """
+  Bundlex compatible implementation of the Membrane.RawVideo struct.
+
+  Currently bundlex does not support tuples in structs, so we have to decompose framerate into two fields.
+  """
+
+  @typedoc """
+  Currently supported formats used to encode the color of every pixel in each video frame.
+  """
+
+  @type pixel_format_t ::
+          :I420 | :I422 | :I444
+
+  @typedoc """
+  Width of single frame in pixels.
+  """
+  @type width_t :: pos_integer()
+
+  @typedoc """
+  Height of single frame in pixels.
+  """
+  @type height_t :: pos_integer()
+
+  @typedoc """
+  Numerator of number of frames per second. To avoid using tuple type,
+  it is described by 2 separate integers number.
+  """
+  @type framerate_num_t :: non_neg_integer
+
+  @typedoc """
+  Denominator of number of frames per second. To avoid using tuple type,
+  it is described by 2 separate integers number. Default value is 1.
+  """
+  @type framerate_den_t :: pos_integer
+
+  @type t :: %__MODULE__{
+          width: width_t(),
+          height: height_t(),
+          pixel_format: pixel_format_t(),
+          framerate_num: framerate_num_t(),
+          framerate_den: framerate_den_t(),
+          aligned: boolean()
+        }
```

Comment:
There's no point in duplicating the types from `RawVideo` here, you can use them in the struct type definition
```suggestion
    @typedoc """
  A numerator of the number of frames per second. To avoid using tuple type,
  it is described by 2 separate integers number.
  """
  @type framerate_num_t :: non_neg_integer

  @typedoc """
  The denominator of the number of frames per second. To avoid using tuple type,
  it is described by 2 separate integers number. The default value is 1.
  """
  @type framerate_den_t :: pos_integer

  @type t :: %__MODULE__{
          width: RawVideo.width_t(),
          height: RawVideo.height_t(),
          pixel_format: RawVideo.pixel_format_t(),
          framerate_num: framerate_num_t(),
          framerate_den: framerate_den_t(),
          aligned: RawVideo.aligned_t()
        }
```

Reply:
Unifex does not support all `RawVideo`s pixel formats. `pixel_format_t` should be kept